### ISSUE TITLE
Option to close saved page

### ISSUE THREAD

**Issue by nicosomb** [wallabag/wallabagger#174]
I wonder if an option to close the current tab after saving it could be a good idea. What do you think?

**Comment by Rurik19** [comment]
yeah, it is definitely good idea. and if it is the last tab = close browser :smile:

**Comment by jtagcat** [comment]
Many other extensions check if it's the last tab, and create a new tab before closing the last one.

My main browser profile takes about a minute to startup on the laptop, and ~10s on the desktop. It's a nuisance if an extension closed the browser.
